test/e2e/provisioning/testdata/machinedeployment-vsphere.yaml

```diff
@@ -30,12 +30,12 @@ spec:
             username: '<< VSPHERE_USERNAME >>'
             vsphereURL: '<< VSPHERE_ADDRESS >>'
             datacenter: 'Hamburg'
-            folder: '/Hamburg/vm/Kubermatic-ci'
+            folder: '/Hamburg/vm/Kubermatic-dev'
             password: << VSPHERE_PASSWORD >>
             # example: 'https://your-vcenter:8443'. '/sdk' gets appended automatically
-            cluster: Kubermatic
+            cluster: vcenter.dc.k8c.io
```

Comment:
Does a cluster with this name really exists in our vCenter?

Reply:
No, I tried copying something from KKP because I can't access vSphere.

Reply:
Everything should be there in vault to access it.

Reply:
It's what KKP also successfully uses for its e2e tests: https://github.com/kubermatic/kubermatic/blob/main/hack/ci/testdata/seed.yaml#L104

Reply:
Are you sure about that? Cluster is only used when VM anti-affinity is enabled for machines. Does KKP also test that feature?

Reply:
![Screenshot 2024-08-26 at 14 54 52](https://github.com/user-attachments/assets/58468f0a-c65e-4eda-b8d1-98d3ee850208)

This is the response that I get for clusters using govc. But if it's already working for you and you have tested it, as it seems, then we are all good here :D

Reply:
The value in Vault is `Kubermatic`, I'm trying with that at the moment. Originally I took the value from the file that Christoph shared, but that cluster doesn't exist in vSphere, I'm not sure what's that field exactly doing.

Reply:
It also doesn't work with `Kubermatic`. I found `vSAN Cluster` in the UI as well, let me try with that.